addons/binding/org.openhab.binding.geofence/README.md

```diff
@@ -0,0 +1,160 @@
+# Geofence Binding
+
+This binding allows you to connect mobile GPS tracking applications to an openHAB installation and detect 
+presence based on GPS location reports. Currently two applications are supported:
+
+* [OwnTracks](https://owntracks.org/booklet/) - iOS, Android
+* [GPSLogger](https://gpslogger.app/) - Android
+
+GPS location reports are sent to openHAB using HTTP protocol. Please be aware that this communication uses the public network
+so make sure your openHAB installation is [secured](https://www.openhab.org/docs/installation/security.html#encrypted-communication)
+and you configured HTTP**S** access in tracking applications.
+
+The binding can process two message types received from trackers:
+
+* **Location** - This is a simple location report with coordinates extended with some extra information about the tracker (e.g. device battery level). [OwnTracks, GPSLogger]
+* **Transition** - This report is based on regions defined in tracker application. A message is sent every time the device enters or leaves a region. [OwnTracks only]
+
+## Regions
+
+Regions are predefined waypoints with a center location and a radius. The binding defines two region types:
+
+* **External** - region is defined on the tracker device. These regions are usually user specific (e.g. family members can have their own waypoints like school or work) and the tracker device is responsible to send transition messages related to external regions. [supported by OwnTracks]
+* **Internal** - defined centrally in the binding config which regions are shared between all users connected to the openHAB installation. For these regions it is the binding's responsibility to fire the entering or leaving events (the distance of the tracker device is calculated after each location report). The best example for this region type is Home. The openHAB installation location can be set on **System/Regional Settings** configuration screen. This location is interpreted by the binding as the **primary region**. [supported by both OwnTracks and GPSLogger]
+
+## Configuration
+
+### Binding in openHAB
+
+![Image](./docs/binding_config.png)
+After installing the binding the main configuration task is to define the **Internal** regions. 
+There is one primary region which is the location of the openHAB installation. 
+The dialog above shows the details of this primary region configuration where the:
+
+* **Name** - is the name of the region
```

Comment:
Better move those to a table and also add a column with the technical parameter name (and a column with the default values).
Note that those values can be specified in a `services/<bindingid>.cfg` file as well.